src/spatio_temporal_voxel_grid.cpp

```diff
@@ -412,7 +417,15 @@ openvdb::Vec3d SpatioTemporalVoxelGrid::IndexToWorld(
 /*****************************************************************************/
 {
   // Applies tranform stored in getTransform.
-  return _grid->indexToWorld(coord);
+  openvdb::Vec3d pose_world =  _grid->indexToWorld(coord);
```

Comment:
This is not the approved solution in https://github.com/SteveMacenski/spatio_temporal_voxel_layer/commit/7bf1e12aea72269cda30a483ddeefbe7a333e592.

Reply:
Why not use that one? There's no reason to mess with the internals of this if we can simply apply an offset when converting to grid coordinates for the costmap

Reply:
Because https://github.com/SteveMacenski/spatio_temporal_voxel_layer/commit/7bf1e12aea72269cda30a483ddeefbe7a333e592 did not work fine.
![image](https://user-images.githubusercontent.com/24326888/154214179-707b5f21-ae75-43ea-bfe8-27580ba6c8d9.png)

I noticed that if the pose_world represents the vertex of the grid, the grid occupation depends on different quadrants. I guess it's caused by the precision of double value, and there is deviation when converting to grid coordinates for the costmap. So I apply an offset to all points so that the pose_world represents the center of the grid. It works fine when costmap resolution >= voxel_size.

when costmap resolution(0.02) < voxel_size(0.04): 
![image](https://user-images.githubusercontent.com/24326888/154190590-a47d8b60-0f4f-4afd-9b9f-f8b86e21dc16.png)

Reply:
OK, let me take a look at your solution then next week if we should be using this instead so I can update all of the appropriate locations in other branches as well.